addons/mail/static/src/models/thread/thread.js

```diff
@@ -1363,6 +1363,46 @@ function factory(dependencies) {
             return [['replace', this.activities.filter(activity => activity.state === 'overdue')]];
         }
 
+        /**
+         * Current partner seen infos.
+         *
+         * @private
+         * @returns {[mail.partner_seen_infos]}
+         */
+        _computeCurrentPartnerSeenInfos() {
+            if (!this.messaging) {
+                return [['unlink-all']];
+            }
+            if (!this.messaging.currentPartner) {
+                return [['unlink-all']];
+            }
+            return [['replace', this.partnerSeenInfos.filter(partnerSeenInfo => partnerSeenInfo.partner === this.messaging.currentPartner)]]
+        }
+
+        /**
+         * Not a real field, used to trigger `markAsFetch` when new message(s) is(are) fetched
+         *
+         * @private
+         */
+        _computeShouldBeSetAsFetched() {
+            if (this.model !== 'mail.channel') {
+                return;
+            }
+            if (this.channel_type === 'channel') {
```

Comment:
I just realized, do we not exclude those (all of the seen conditions) from "group" channels? Because they will make a big issue too, we should definitely exclude them everywhere.

Reply:
It's the same condition as in the server so I don't understand .. ?

Reply:
I think the condition on the server is not wide enough then but let's double check it in another task instead.

Reply:
Ok I'm crazy, "group" is no "channel_type" but "public" enum. Forget about this